### ISSUE TITLE
Tooltip accessibility

### ISSUE THREAD

**Issue by jgiardino** [patternfly/patternfly-next#456]
This issue is to capture the accessibility requirements related to tooltips. As part of this issue:

- aria attributes should be added that enable screen reader access to tooltip text
- screen reader tests should happen to confirm that support is provided as expected. Testing can be completed in the latest versions of JAWS using Chrome, NVDA using FF, and VO on mac.

Depending on the solution, this might be dependent on #431

Resources related to creating accessible tooltips:

- https://www.w3.org/TR/wai-aria-practices-1.1/#tooltip
- https://inclusive-components.design/tooltips-toggletips/
- Also note that tooltips are expected to be dismissible via Esc key per a new [WCAG item related to hover/focus](https://www.w3.org/WAI/WCAG21/Understanding/content-on-hover-or-focus.html)

**Comment by jgiardino** [comment]
This part is done:
- aria attributes should be added that enable screen reader access to tooltip text


The rest is captured in https://github.com/patternfly/patternfly-react/issues/789
